opa/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,21 @@
+name: opa
+files:
+- name: opa.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
```

Comment:
Since this is an `OpenMetricsBaseCheck`, you can use `template: init_config/openmetrics` instead of `init_config/default`

Reply:
👍